org.apache.zeppelin.helium.HeliumTestApplication.run : r3 = $r14 >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r15 = new org.apache.zeppelin.helium.ApplicationException
org.apache.zeppelin.helium.HeliumTestApplication.run : $r15 = new org.apache.zeppelin.helium.ApplicationException >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : specialinvoke $r15.<org.apache.zeppelin.helium.ApplicationException: void <init>(java.lang.Exception)>(r3)
org.apache.zeppelin.helium.HeliumTestApplication.run : specialinvoke $r15.<org.apache.zeppelin.helium.ApplicationException: void <init>(java.lang.Exception)>(r3) >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : throw $r15
org.apache.zeppelin.helium.HeliumTestApplication.run : $r4 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r5 = $r4.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out>
org.apache.zeppelin.helium.HeliumTestApplication.run : $r4 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r5 = $r4.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r5.<org.apache.zeppelin.interpreter.InterpreterOutput: void clear()>()
org.apache.zeppelin.helium.HeliumTestApplication.run : $r5 = $r4.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r5.<org.apache.zeppelin.interpreter.InterpreterOutput: void clear()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r6 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>()
org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r5.<org.apache.zeppelin.interpreter.InterpreterOutput: void clear()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r6 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r7 = $r6.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out>
org.apache.zeppelin.helium.HeliumTestApplication.run : $r6 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r7 = $r6.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r2 = new java.lang.StringBuilder
org.apache.zeppelin.helium.HeliumTestApplication.run : $r7 = $r6.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r2 = new java.lang.StringBuilder >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : specialinvoke $r2.<java.lang.StringBuilder: void <init>()>()
org.apache.zeppelin.helium.HeliumTestApplication.run : $r2 = new java.lang.StringBuilder >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : specialinvoke $r2.<java.lang.StringBuilder: void <init>()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r8 = virtualinvoke $r2.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("Hello world ")
org.apache.zeppelin.helium.HeliumTestApplication.run : specialinvoke $r2.<java.lang.StringBuilder: void <init>()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r8 = virtualinvoke $r2.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("Hello world ") >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r9 = r0.<org.apache.zeppelin.helium.HeliumTestApplication: java.util.concurrent.atomic.AtomicInteger numRun>
org.apache.zeppelin.helium.HeliumTestApplication.run : $r8 = virtualinvoke $r2.<java.lang.StringBuilder: java.lang.StringBuilder append(java.lang.String)>("Hello world ") >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r9 = r0.<org.apache.zeppelin.helium.HeliumTestApplication: java.util.concurrent.atomic.AtomicInteger numRun> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $i0 = virtualinvoke $r9.<java.util.concurrent.atomic.AtomicInteger: int incrementAndGet()>()
org.apache.zeppelin.helium.HeliumTestApplication.run : $r9 = r0.<org.apache.zeppelin.helium.HeliumTestApplication: java.util.concurrent.atomic.AtomicInteger numRun> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $i0 = virtualinvoke $r9.<java.util.concurrent.atomic.AtomicInteger: int incrementAndGet()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r10 = virtualinvoke $r8.<java.lang.StringBuilder: java.lang.StringBuilder append(int)>($i0)
org.apache.zeppelin.helium.HeliumTestApplication.run : $i0 = virtualinvoke $r9.<java.util.concurrent.atomic.AtomicInteger: int incrementAndGet()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r10 = virtualinvoke $r8.<java.lang.StringBuilder: java.lang.StringBuilder append(int)>($i0) >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r11 = virtualinvoke $r10.<java.lang.StringBuilder: java.lang.String toString()>()
org.apache.zeppelin.helium.HeliumTestApplication.run : $r10 = virtualinvoke $r8.<java.lang.StringBuilder: java.lang.StringBuilder append(int)>($i0) >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r11 = virtualinvoke $r10.<java.lang.StringBuilder: java.lang.String toString()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r7.<org.apache.zeppelin.interpreter.InterpreterOutput: void write(java.lang.String)>($r11)
org.apache.zeppelin.helium.HeliumTestApplication.run : $r11 = virtualinvoke $r10.<java.lang.StringBuilder: java.lang.String toString()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r7.<org.apache.zeppelin.interpreter.InterpreterOutput: void write(java.lang.String)>($r11) >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r12 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>()
org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r7.<org.apache.zeppelin.interpreter.InterpreterOutput: void write(java.lang.String)>($r11) >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r12 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r13 = $r12.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out>
org.apache.zeppelin.helium.HeliumTestApplication.run : $r12 = virtualinvoke r0.<org.apache.zeppelin.helium.HeliumTestApplication: org.apache.zeppelin.helium.ApplicationContext context()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : $r13 = $r12.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r13.<org.apache.zeppelin.interpreter.InterpreterOutput: void flush()>()
org.apache.zeppelin.helium.HeliumTestApplication.run : $r13 = $r12.<org.apache.zeppelin.helium.ApplicationContext: org.apache.zeppelin.interpreter.InterpreterOutput out> >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r13.<org.apache.zeppelin.interpreter.InterpreterOutput: void flush()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : goto [?= return]
org.apache.zeppelin.helium.HeliumTestApplication.run : virtualinvoke $r13.<org.apache.zeppelin.interpreter.InterpreterOutput: void flush()>() >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : $r14 := @caughtexception
org.apache.zeppelin.helium.HeliumTestApplication.run : goto [?= return] >>>>>>>> org.apache.zeppelin.helium.HeliumTestApplication.run : return
